src/apicalls/supabaseCalls/itineraryRequestSupabaseCalls.ts

```
@@ -8,5 +8,7 @@ export const getAllItineraryRequest = async () => {
   if (error) {
     console.log('couldn\'t do it: itinerary edit/delete request,', error);
   }
-  return { data, error };
+  if (data) {
```

we will actually want to return the error as well, for toast messages that will come up if there is an error/issue

---

updated to return data and error